proto/invocation.proto

```diff
@@ -170,6 +170,12 @@ message InvocationQuery {
 
   // The ROLE metadata set on the build.
   string role = 6;
+
+  // The timestamp on or after which the build was last updated (inclusive).
+  google.protobuf.Timestamp updated_timestamp_range_start = 7;
+
+  // The timestamp up to which the build was last updated (exclusive).
+  google.protobuf.Timestamp updated_timestamp_range_end = 8;
```

Comment:
(idk what this looks like on the client side but another way to do this would be to have a timestamp + a duration)

What about calling these fields:
updated_after and updated_before?

Reply:
Those names SG (Done) -- it's more ergonomic for the client to use start/end dates so will keep as timestamps